### ISSUE TITLE
[Docs] MovR Tutorial for Hibernate

### ISSUE THREAD

**Issue by jseldess** [cockroachdb/docs#6265]
Background: https://airtable.com/tblD3oZPLJgGhCmch/viw1DKmbKhg2MIECH/recKIdltSmlXwD4LG

Description: CockroachDB now provides application code and documentation for using MovR with Hibernate

Each ORM has a simple getting started guide. This is great to unblock basic usage by doesn't provide extensive guidance to application developers for how to use CockroachDB with a robust application.

Team: Andrew Woods, Rafi Shamim, Eric Harmeling

Github Tracking Issue:

**Comment by ericharmeling** [comment]
This isn't happening for 20.1. 

**Note:** We have a a separate issue for the general "Build an App" tutorial for this ORM.
